mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdDAO.java

```diff
@@ -243,7 +245,16 @@ private ComposedMessageIdWithMetaData fromRowToComposedMessageIdWithFlags(Row ro
                         messageIdFactory.of(row.getUUID(MESSAGE_ID)),
                         MessageUid.of(row.getLong(IMAP_UID))))
                 .flags(new FlagsExtractor(row).getFlags())
-                .modSeq(row.getLong(MOD_SEQ))
+                .modSeq(readModSeq(row))
                 .build();
     }
+
+    private long readModSeq(Row row) {
+        long oldModSeq = row.getLong(MOD_SEQ);
```

Comment:
- I don't insert it
 - It will not be set as I take in priority the value in MOD_SEQ_SET